cmake/CMakeLists.txt

```diff
@@ -179,6 +180,10 @@ if (onnxruntime_ENABLE_LANGUAGE_INTEROP_OPS)
   add_compile_definitions(ENABLE_LANGUAGE_INTEROP_OPS)
 endif()
 
+if (onnxruntime_ENABLE_TRAINING)
+  add_compile_definitions(ENABLE_TRAINING_OPS)
```

Comment:
could you please move this to line 1255, there is already a if block for onnxruntime_ENABLE_TRAINING